tests/test-vandv/src/main/java/mil/emp3/test/emp3vv/containers/dialogs/UpdateFeatureDialog.java

```diff
@@ -104,12 +108,17 @@ protected void resetAddChildrenList() {
 
     @Override
     protected void setupOptional(View view) {
-
+        // Color field only needs to appear for MilStdSymbols.
+        if(this.me instanceof MilStdSymbol) {
```

Comment:
What is `this.me`?  It seems really bizzarre....

Reply:
this.me is the actual feature. It is bizarre.

Reply:
I am torn on what to do with `test-vandv` classes because I feel like they need a major overhaul but that is more work than I care do have us do at this point.  For now I think I will make an epic and we can put smaller sub-tasks underneath. I will discuss with @rsupnekar at some point soon.